pkg/reconciler/route/resources/certificate.go

```diff
@@ -58,7 +59,7 @@ func MakeCertificate(owner kmeta.OwnerRefableAccessor, ownerLabelKey string, dns
 // domainTagMap is an one-to-one mapping between domain and tag, for major domain (tag-less),
 // the value is an empty string
```

Comment:
Add documentation for `domain` here, since it's not clear how it relates to `domainTagMap`?